src/main/scala/com/mwz/sonar/scala/scalastyle/ScalastyleSensor.scala

```diff
@@ -37,7 +38,10 @@ import scala.collection.immutable.Seq
 /**
  * Main sensor for executing Scalastyle analysis.
  */
-final class ScalastyleSensor(qualityProfile: QualityProfile) extends Sensor {
+final class ScalastyleSensor(
+  qualityProfile: QualityProfile,
+  scalastyleChecker: ScalastyleChecker[FileSpec]
```

Comment:
Sorry. I forgot to mention this in the previous review.
Are you sure this will work?, How can **SonarQube** inject a new instance of `ScalastyleChecker` ?
I believed it would need the `@ScannerSide` annotation, like [here](https://github.com/mwz/sonar-scala/blob/master/src/main/scala/com/mwz/sonar/scala/scapegoat/ScapegoatReportParser.scala#L31)

Reply:
That's one of the things I wanted to check - I'll let you know if it works.

Reply:
ok

Reply:
You were right, I fixed it in 1f05525.